docs/CONTRIBUTING.rst

```diff
@@ -1,5 +1,89 @@
+
+
+Contributing to the project
+===========================
+
+We strongly advise to use `Virtualenv <https://virtualenv.pypa.io/en/stable/>`_ to install Cekit. Please consult your package manager for the correct package name.
```

Comment:
I would probably add some "prerequisities" or similar section here that would describe steps required to start the development process.

Reply:
Rephased.